sdk/examples/wallet/logger.rs

```diff
@@ -38,18 +39,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     // Restore a wallet
     let client_options = ClientOptions::new().with_node(&std::env::var("NODE_URL").unwrap())?;
-    let secret_manager = MnemonicSecretManager::try_from_mnemonic(std::env::var("MNEMONIC").unwrap())?;
-    let wallet = Wallet::builder()
-        .with_secret_manager(SecretManager::Mnemonic(secret_manager))
```

Comment:
This shouldn't be removed?

Reply:
I think I have some time to review that code myself because I just can't remember why I did some of those changes.

Reply:
fixed :see_no_evil: